dts/bindings/timer/nordic,nrf-timer.yaml

```diff
@@ -24,10 +24,15 @@ properties:
   interrupts:
     required: true
 
+  max-frequency:
+    type: int
+    default: 16
```

Comment:
default value needs justification in the description

Reply:
Thanks @decsny . Done